src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,171 @@
+/**
+ * This class performs the commands in Duke.
+ *
+ * @author Megan Wee Rui En
+ * @version CS2103T AY21/22 Semester 1
+ */
+
+package duke.command;
+
+import duke.exceptions.EmptyDescriptionException;
+import duke.exceptions.NotDoneRightException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+import duke.ui.Ui;
+
+import java.io.IOException;
+import java.time.LocalDate;
+
+public class Command {
+
+    private final Storage storage;
+    private final Ui ui;
+
+    public Command(Storage storage, Ui ui) {
+        this.storage = storage;
+        this.ui = ui;
+    }
+
+    /**
+     * Adds a deadline to the task list.
+     *
+     * @param user_command The command inputted from the user
+     * @param cmd The type of task added to the task list
+     * @param taskList The task list which contains the tasks
+     * @throws EmptyDescriptionException
+     * @throws IOException
+     */
+    public void addDeadline(String user_command, String cmd, TaskList taskList)
```

Comment:
Would it be better if `user_command` is written in lowerUpperCase instead as with all other variables?